infrastructure/deploy.sh

```diff
@@ -0,0 +1,117 @@
+#!/bin/bash -e
+
+# This script must be run from /infrastructure!
+
+# Make our IP address known to terraform.
+source ../common.sh
+export TF_VAR_host_ip=`wget -qO- http://ipecho.net/plain ; echo`
```

Comment:
This is an SPoF if we're using this in prod. It might be possible to do with with system DNS tools rather than a third party web service.